modules/environment/main.tf

```diff
@@ -1,34 +1,20 @@
-module "environment_context" {
-  source = "../context"
-
-  enabled = true
-
-  context_name          = var.stack_config_name
-  environment_variables = var.environment_values
-}
-
-module "components" {
+module "stacks" {
   source = "../stack"
 
-  for_each = {
-    for k, v in var.components : "${var.stack_config_name}-${k}" => merge({ "component" : k }, v)
-  }
+  for_each = { for k, v in var.components : "${var.environment_values.environment}-${var.environment_values.stage}-${k}" => merge({ "component" : k }, v) }
```

Comment:
Hrmm... seems like `${var.environment_values.environment}-${var.environment_values.stage}` should be replaced by stack, no? 

`environment` and `stage` are technically optional.

Reply:
Also, prefer using `format` function instead.

See https://github.com/cloudposse/terraform-yaml-stack-config/pull/4/files#diff-4964e36621e70b8e1206fac66ce8ffdf2fb3471ca6db94658e64cab6dddd0468R11